Library/src/Multiplayer/Script/ComponentScriptInterface.cpp

```diff
@@ -64,6 +64,24 @@ int64_t ComponentScriptInterface::GetComponentType() const
 	return (int64_t) ComponentType::Invalid;
 }
 
+void ComponentScriptInterface::SetComponentName(std::string name)
+{
+	if (Component)
```

Comment:
Should we emit an error log if component == nullptr for the setter and getter?

Reply:
We currently dont do this for any other script component props

Reply:
Understood. Do you think we should?

Reply:
We should further investigate this by understanding the circumstances where the COmponent would be null to find the best method to handle this

Reply:
Makes sense. Thanks!